contracts/GovHub.template

```diff
@@ -243,7 +244,7 @@ contract GovHub is System, IParamSubscriber {
 
   /// Execute the proposal
   /// @param proposalId The proposal Id
-  function execute(uint256 proposalId) public payable onlyInit onlyIfSuccessfulProposal(proposalId) {
+  function execute(uint256 proposalId) public payable nonReentrant onlyInit onlyIfSuccessfulProposal(proposalId) { //@openissue
```

Comment:
I would keep the guard here although execute() only uses platform contracts as targets for additional safety (future changes?). also note that the target calls will never loop-back to execute() so no DoS danger here.